include/openssl/x509.h.in

```diff
@@ -1076,6 +1085,84 @@ int X509_PUBKEY_get0_param(ASN1_OBJECT **ppkalg,
                            X509_ALGOR **pa, const X509_PUBKEY *pub);
 int X509_PUBKEY_eq(const X509_PUBKEY *a, const X509_PUBKEY *b);
 
+DECLARE_ASN1_FUNCTIONS(X509_ACERT)
+DECLARE_ASN1_FUNCTIONS(X509_ACERT_INFO)
+DECLARE_ASN1_FUNCTIONS(OBJECT_DIGEST_INFO)
+DECLARE_ASN1_FUNCTIONS(ISSUER_SERIAL)
+
+#define X509_ACERT_VERSION_2 1
+
+const X509_NAME *X509_ACERT_get0_holder_entityName(const X509_ACERT *x);
+ISSUER_SERIAL *X509_ACERT_get0_holder_baseCertId(const X509_ACERT *x);
+OBJECT_DIGEST_INFO * X509_ACERT_get0_holder_digest(const X509_ACERT *x);
+const X509_NAME *X509_ACERT_get0_issuerName(const X509_ACERT *x);
+long X509_ACERT_get_version(const X509_ACERT *x);
+void X509_ACERT_get0_signature(const X509_ACERT *x,
+         const ASN1_BIT_STRING **psig, const X509_ALGOR **palg);
+X509_ALGOR *X509_ACERT_get0_info_signature(const X509_ACERT *x);
+ASN1_INTEGER *X509_ACERT_get_serialNumber(X509_ACERT *x);
+const ASN1_TIME *X509_ACERT_get0_notBefore(const X509_ACERT *x);
+const ASN1_TIME *X509_ACERT_get0_notAfter(const X509_ACERT *x);
+int X509_ACERT_get_attr_count(const X509_ACERT *x);
+int X509_ACERT_get_attr_by_NID(const X509_ACERT *x, int nid, int lastpos);
+int X509_ACERT_get_attr_by_OBJ(const X509_ACERT *x, const ASN1_OBJECT *obj, int lastpos);
+X509_ATTRIBUTE *X509_ACERT_get_attr(const X509_ACERT *x, int loc);
+X509_ATTRIBUTE *X509_ACERT_delete_attr(X509_ACERT *x, int loc);
+ASN1_BIT_STRING *X509_ACERT_get0_issuerUID(X509_ACERT *x);
+const STACK_OF(X509_EXTENSION) *X509_ACERT_get0_extensions(const X509_ACERT *x);
+int X509_ACERT_print(BIO *bp, X509_ACERT *x);
+int X509_ACERT_print_ex(BIO *bp, X509_ACERT *x, unsigned long nmflags,
+                      unsigned long cflag);
```

Comment:
nit : please align the arguments between the two lines like this :
```
int X509_ACERT_print_ex(BIO *bp, X509_ACERT *x, unsigned long nmflags,
                        unsigned long cflag);
```
(not sure of the display result of my example.)